README.md

```diff
@@ -31,14 +30,16 @@ npm takes care of the rest of our dependencies.
     ``` npm install ```
 
 ## Creating a dotenv file
+
 The app needs to point to the hostname where the backend expects to receive HTTP requests. This has to be set before building the client.
 
 In order to set up all that, create a file at the location /var/www/pwa-prototype-reporting/.env . Use the following contents as an example:
 
 ```
-BACKEND_URL=http://ushahidi-backend
-   client_secret='35e7f0bca957836d05ca0492211b0ac707671261'
+baseUrl=http://ushahidi-backend.io
```

Comment:
can you change this (and all references to it) to base_url so it's consistent with the other variables ? (all the other variables here use snake_case but this uses lowerCamelCase